### PR TITLE
remove sigstore config with trust delete

### LINKED ISSUE #647: trust delete should have a "rm sigstore" option
Deleting trust removes it from the policy.json but leaves the registries.d config around. That's the correct default behavior but we should allow a user to cleanup the environment.

Not sure what to call this boolean flag...
```
$ atomic trust delete POLICY_NAME --sigstore
```

### PR DESCRIPTION
**Pull request by aweiteka**
resolves #647 and #694

### CODE REVIEW

File: docs/atomic-trust.1.md [hunk 1]
```diff
@@ -72,6 +72,11 @@ testing.
     **atomic**: OpenShift-based Atomic Registry API
     **local**: Local filesystem path
 
+# delete OPTIONS
+
+**--sigstore**
```
Comment (rhatdan) [inline docs/atomic-trust.1.md]: Default is true.

File: docs/atomic-trust.1.md [hunk 2]
```diff
@@ -98,6 +103,10 @@ Remove a trust scope
 
     atomic trust delete docker.io
 
+Remove a trust scope but retain the sigstore configuration
+
+    atomic trust delete docker.io --sigstore
```
Comment (rhatdan) [inline docs/atomic-trust.1.md]: I think you would need to say

 atomic trust delete docker.io --sigstore=false

Comment (aweiteka) [inline docs/atomic-trust.1.md]: This is a boolean flag so it fails when a value is passed in.

```
$ atomic trust delete --sigstore=false
atomic trust delete: argument --sigstore: ignored explicit argument 'false'
Try 'atomic trust delete --help' for more information.
```

Comment (aweiteka) [inline docs/atomic-trust.1.md]: Good idea.

**Comment by aweiteka** [comment]
@rhatdan all set.

File: Atomic/trust.py [hunk 3]
```diff
@@ -63,6 +63,8 @@ def cli(subparser):
     deletep.add_argument("--sigstoretype", dest="sigstoretype", default="web",
                          choices=['atomic', 'local', 'web'],
                          help=sigstore_help)
+    deletep.add_argument("--save-sigstore", dest="savesigstore", default=True, action="store_false",
```
Comment (aweiteka) [inline Atomic/trust.py]: The only way I see to make this work is to use `default=True, action="store_false"`. Any other combination I've tested does not produce the behavior we document and expect.

**Pull request merged**